NextGenConnect/JSON to HL7v2.xml

```diff
@@ -340,9 +345,16 @@ tmp[&apos;RXA&apos;][&apos;RXA.9&apos;] = 00;
 
 
 //RXA-10 Administering Provider 
+try{
+	tmp[&apos;RXA&apos;][&apos;RXA.10&apos;][&apos;RXA.10.1&apos;] = actor[&apos;identifier&apos;][0][&apos;value&apos;];
+} catch (e) {
+	//TypeError means the Practitioner resource has no non-FHIR identifiers
+	if (!(e instanceof TypeError)) throw e;
+	tmp[&apos;RXA&apos;][&apos;RXA.10&apos;][&apos;RXA.10.1&apos;] = actor[&apos;id&apos;];
+}
 
-tmp[&apos;RXA&apos;][&apos;RXA.10&apos;][&apos;RXA.10.1&apos;] = actor[&apos;name&apos;][0][&apos;family&apos;];
-tmp[&apos;RXA&apos;][&apos;RXA.10&apos;][&apos;RXA.10.2&apos;] = actor[&apos;name&apos;][0][&apos;given&apos;][0];
+tmp[&apos;RXA&apos;][&apos;RXA.10&apos;][&apos;RXA.10.2&apos;] = actor[&apos;name&apos;][0][&apos;family&apos;];
```

Comment:
Mirth throws an error here because the practitioner resource doesnt have a name

Reply:
I've put the names in the try/catch, so it should proceed if the provider doesn't have a name. We'll have to think about validating the HL7 message at some point, although probably not as part of this PR.

Reply:
I've put the names in the try/catch, so it should proceed if the provider doesn't have a name. We'll have to think about validating the HL7 message at some point, although probably not as part of this PR.